### ISSUE TITLE
Fix file selector width

### ISSUE THREAD

**Issue by dfguerrerom** [12rambau/sepal_ui#478]
Similarly to the https://github.com/sepal-contrib/weplan/issues/8 issue, the `FileSelector` widget is expanded and shrunk depending on its content, set a min-max width property will help on this, also, we can truncate those long names to fit in the given size, what do you think?

**Comment by 12rambau** [comment]
So far I never experienced any issues with the FileSelector width as I'm usually using it in a fixed sized environment. I would say maybe better to leave the developer customize it on initialization using kwargs  as it's a very basic component. We could eventually document it somwhere. 

for the name I fail to find a nice idea without loosing information, I suggest we open a question on stackExchange UI I'm sure they'll come up with a nice idea

**Comment by 12rambau** [comment]
Ok back on this issue, I think I didn't understand what you were talking about the first time. 
You would like to fix the menu size right ? 
Do you have any fixed value in mind ?

**Comment by dfguerrerom** [comment]
Yes, more or less, perhaps having it as some standard width (600?, there should be a standard for that), but the thing was mainly because normally each folder has a different  file lenghts, and every time the menu width will be compressed or extended depending on the content, I;ve have had some 100char files and this is not nice to see.

**Comment by 12rambau** [comment]
I have a time series stored somewhere and the naming convetion is terrible that should do the trick for testing
